tools/scripts/seed/seedAuthUser.js

```diff
@@ -64,6 +64,7 @@ MongoClient.connect(MONGOHQ_URL, { useNewUrlParser: true }, function(
         isApisMicroservicesCert: false,
         isInfosecQaCert: false,
         is2018FullStackCert: false,
+        isSciPyCert: false,
```

Comment:
Missing Certs?

Reply:
Yes, we appear to be missing the data analysis and machine learning certifications here, too.